ops/src/repo_version.rs

```diff
@@ -285,6 +286,7 @@ impl worker::Runner<RepoVersionRequest> for Runner {
                                 &req.repo,
                                 &req.branch,
                                 &req.commits,
+                                vec![req.repo.html_url.to_string()], // Todo(NTD) - when does this get called???
```

Comment:
this issues a message to the channel in case it encounters an error running the version script. 

like, when you merge in a PR into main, it comments on jira with the version number, etc.